web/src/lib/components/asset-viewer/activity-viewer.svelte

```diff
@@ -48,32 +48,30 @@
   };
 
   interface Props {
-    reactions: ActivityResponseDto[];
     user: UserResponseDto;
     assetId?: string | undefined;
     albumId: string;
     assetType?: AssetTypeEnum | undefined;
     albumOwnerId: string;
     disabled: boolean;
     isLiked: ActivityResponseDto | null;
-    onDeleteComment: () => void;
-    onDeleteLike: () => void;
-    onAddComment: () => void;
+    onDeleteComment?: () => void;
+    onDeleteLike?: () => void;
+    onAddComment?: () => void;
     onClose: () => void;
   }
 
   let {
-    reactions = $bindable(),
     user,
     assetId = undefined,
     albumId,
     assetType = undefined,
     albumOwnerId,
     disabled,
     isLiked,
-    onDeleteComment,
-    onDeleteLike,
-    onAddComment,
+    onDeleteComment = () => {},
+    onDeleteLike = () => {},
+    onAddComment = () => {},
```

Comment:
you are already importing and using the activityManager in this file. What is the point of calling a callback that all it does it call the activity manager itself?